app/qml/ActiveLayerPanel.qml

```diff
@@ -22,6 +42,22 @@ Drawer {
         color: InputStyle.clrPanelMain
     }
 
+    Item {
```

Comment:
do we need this sub-item to define states? can't be put the states to the root ActiveLayerPanel item?

Reply:
The panel itself i a drawer which is inherited from Popup which is inherited from QtObject. None of those supports state/states property.